pkgs/applications/misc/ausweisapp2/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, mkDerivation, fetchFromGitHub, cmake, pkgconfig, pcsclite, qtsvg, qttools, qtwebsockets
+, qtquickcontrols2, qtgraphicaleffects }:
+
+mkDerivation rec {
+  pname = "ausweisapp2";
```

Comment:
I think we should stick to the exact upstream name, including casing (here and for the attrname in all-packages).

Reply:
Right, the changed casing also bothers me with many Python packages, so let's keep it equal.

Pushed a new version with the correct name.